include/libfastboot/keybox_provision.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _KEYBOX_PROVISION_H_
+#define _KEYBOX_PROVISION_H_
+
+#define ATTKB_META_BASE_ADDRESS	       10
+#define ATTKB_META_SIGNATURE	       "BARA"
+#define ATTKB_META_SIGNATURE_LENGTH    (sizeof(ATTKB_META_SIGNATURE) - 1)
```

Comment:
is this a legal usage sizeof(ATTKB_META_SIGNATURE) ? 
sizeof("BARA")? what's the expected output? just be curious:)